superset-frontend/src/explore/components/controls/DatasourceControl/index.jsx

```diff
@@ -200,7 +200,8 @@ class DatasourceControl extends React.PureComponent {
     const { user } = this.props;
     const allowEdit =
       datasource.owners.map(o => o.id).includes(user.userId) ||
-      isUserAdmin(user);
+      datasource.owners.map(o => o.value).includes(user.userId);
```

Comment:
couldn't you write this in without the ||

`o => o.id || o.value == user.userId `

Reply:
don't think this is the right way of doing it though.

Reply:
also should this be an || or should you be amending allowEdit with the new values from o.value?